webapi/src/main/scala/org/knora/webapi/util/StringFormatter.scala

```diff
@@ -3020,35 +3021,6 @@ class StringFormatter private(val maybeSettings: Option[SettingsImpl] = None, ma
         s"http://$IriDomain/permissions/$shortcode/$knoraPermissionUuid"
     }
 
-    /**
```

Comment:
these parts seem to be not relevant to the main goal of current PR. Can they be moved to a separate PR?

Reply:
OK, I undid this change in 6e96460 and will move it to another PR. These are just functions that I forgot to delete a long time ago.